src/app/machines/views/MachineList/MachineList.tsx

```diff
@@ -45,15 +45,19 @@ const MachineList = ({
   const dispatch = useDispatch();
   const errors = useSelector(machineSelectors.errors);
   const selectedIDs = useSelector(machineSelectors.selectedIDs);
+  const [currentPage, setCurrentPage] = useState(1);
```

Comment:
Might be worth covering this with tests.

Reply:
Done.